modules/account-lib/src/mpc/curves/secp256k1.ts

```diff
@@ -0,0 +1,56 @@
+import { publicKeyCombine } from 'secp256k1';
+import { bigIntFromBufferBE, bigIntToBufferBE } from '../util';
+import { modulo } from '../../utils/helpers';
+import BaseCurve from './baseCurve';
+import * as secp from '@noble/secp256k1';
+
+const order = secp.CURVE.n;
+
+export class Secp256k1Curve implements BaseCurve {
+  scalarRandom(): bigint {
+    return bigIntFromBufferBE(Buffer.from(secp.utils.randomPrivateKey()));
+  }
+
+  scalarAdd(x: bigint, y: bigint): bigint {
+    return bigIntFromBufferBE(Buffer.from(secp.utils.privateAdd(modulo(x), modulo(y).toString(16))));
+  }
+
+  scalarSub(x: bigint, y: bigint): bigint {
+    return modulo(x - y);
+  }
+
+  scalarMult(x: bigint, y: bigint): bigint {
+    return modulo(x * y);
+  }
+
+  scalarReduce(s: bigint): bigint {
+    return modulo(s);
+  }
+
+  scalarNegate(s: bigint): bigint {
+    return bigIntFromBufferBE(Buffer.from(secp.utils.privateNegate(modulo(s))));
+  }
+
+  scalarInvert(s: bigint): bigint {
+    return bigIntFromBufferBE(Buffer.from(secp.utils.privateInvert(bigIntToBufferBE(modulo(s), 32))));
+  }
+
+  pointAdd(a: bigint, b: bigint): bigint {
+    const pointA = bigIntToBufferBE(a);
+    const pointB = bigIntToBufferBE(b);
+    return bigIntFromBufferBE(Buffer.from(publicKeyCombine([pointA, pointB], true)));
```

Comment:
Let's use 1 crypto library consistently:

```suggestion
    const pointA = secp.Point.fromHex(bigIntToBufferBE(a));
    const pointB = secp.Point.fromHex(bigIntToBufferBE(b));
    return bigIntFromBufferBE(Buffer.from(pointA.add(pointB).toRawBytes(true)));
```